Makefile

```diff
@@ -47,12 +47,18 @@ wait-healthy: ## Wait for the containers to be healthy
 sh: ## Open a shell on the app container
 	make exec command="sh"
 
+tty=1
+ifeq ($(tty), 0)
+	options=-T
+else
+	options=
+endif
```

Comment:
have to do this with a Make conditional rather than a bash one because it's proving difficult to interpolate an `options` variable inside the `docker-compose ...` command without creating a subshell. The `command` check could be dealt with in the same way, but this approach is executed when loading the Makefile rather than when executing particular targets.

Reply:
These are also global. Will substitute with `$(if ...)`